src/components/footer.tsx

```diff
@@ -0,0 +1,14 @@
+import * as classNames from 'classnames'
+import * as React from 'react'
+
+interface Props extends React.HTMLProps<HTMLElement> {
+    title: string
+}
+
+const Footer = ({ title, ...rest }: Props) => (
+    <footer className={classNames('footer')} {...rest}>
```

Comment:
since this is the only class you are adding here, I would probably ditch `classNames` and just do `className="footer"`.